ts-sdk/whirlpool/src/sysvar.ts

```diff
@@ -0,0 +1,28 @@
+import { fetchSysvarRent } from "@solana/sysvars";
```

Comment:
nit: This should be re-exported by @solana/web3.js so you can use that as import